www-apps/radicale/radicale-2.1.11-r1.ebuild

```diff
@@ -79,4 +80,13 @@ python_install_all() {
 pkg_postinst() {
 	einfo "A sample WSGI script has been put into ${ROOT%/}/usr/share/${PN}."
 	einfo "You will also find there an example FastCGI script."
+	if [[ $(stat --format="%U:%G:%a" ${RDIR}) != "${PN}:${PN}:750" ]]
+	then
```

Comment:
This will not work on fbsd, but we do not support that yet. Maybe there is a utility function to do such a check platform independant?